src/hotspot/share/logging/logDecorators.cpp

```diff
@@ -24,6 +24,11 @@
 #include "precompiled.hpp"
 #include "logging/logDecorators.hpp"
 #include "runtime/os.hpp"
+#include "logDecorators.hpp"
+
+const LogLevelType AnyLevel = LogLevelType::NotMentioned;
+#define DEFAULT_DECORATORS \
+  DEFAULT_VALUE(mask_from_decorators(NoDecorators), AnyLevel, LOG_TAGS(jit, inlining))
```

Comment:
As a compiler developer, I agree with the choice of no decorators by default for `jit+inlining`. When this is the only tag selected, there isn't much value in the information provided by the global default decorators.